aria/parser/consumption/modeling.py

```diff
@@ -145,6 +145,24 @@ def consume(self):
         self.context.modeling.instance.validate_capabilities()
 
 
+class FindHosts(Consumer):
```

Comment:
This is just how phases work right now in "the parser". We keep talking about all of this moving into a separate "instantiation" module, but this is what we have for now.